docs/classical_control.ipynb

```diff
@@ -147,7 +158,27 @@
     "    cirq.X(bob).with_classical_controls('A'),\n",
     "    cirq.Z(bob).with_classical_controls('M'),\n",
     ")\n",
-    "print(circuit)"
+    "print(circuit)\n",
+    "\n",
+    "# Simulate the message and teleport circuits for bloch vectors\n",
+    "sim = cirq.Simulator()\n",
+    "message_bloch_vector = cirq.bloch_vector_from_state_vector(\n",
+    "    sim.simulate(message_circuit).final_state_vector, index=0\n",
+    ")\n",
+    "teleport_bloch_vector = cirq.bloch_vector_from_state_vector(\n",
+    "    sim.simulate(message_circuit).final_state_vector, index=2\n",
+    ")\n",
+    "print(f\"Message Qubit State: {message_bloch_vector}\")\n",
+    "print(f\"Teleported Bob's Qubit state :{teleport_bloch_vector}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ca879df5fb2d"
+   },
+   "source": [
+    "This example separately simulated the message qubit after it's construction, and Bob's qubit after teleportation of the message. The fact that the bloch vectors of each respective qubit are the same indicate that the circuit successfully teleported the message circuit's state onto Bob's qubit. "
```

Comment:
`after it's` -> `after its`

`message circuit's state` -> `message qubit's state`